easybuild/easyconfigs/g/Graphviz/Graphviz-5.0.0-GCCcore-11.3.0.eb

```
@@ -0,0 +1,95 @@
+easyblock = 'ConfigureMake'
+
+name = 'Graphviz'
+version = '5.0.0'
+local_pyver_major = '3'
+
+homepage = 'https://www.graphviz.org/'
+description = """Graphviz is open source graph visualization software. Graph visualization
+ is a way of representing structural information as diagrams of
+ abstract graphs and networks. It has important applications in networking,
+ bioinformatics,  software engineering, database and web design, machine learning,
+ and in visual interfaces for other technical domains."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://gitlab.com/graphviz/graphviz/-/archive/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['%(name)s-%(version)s_skip-install-data-hook.patch']
+checksums = [
+    'f8cc576694ae93659f453b6b4a4c2f04572a90cfccdca2d4271f2e7c60bb3d60',  # graphviz-5.0.0.tar.gz
+    '4cafb6eac6f59fa4e90cef6e83c276ad5e4e24b10992b18d4b51c6d28dfe4ca1',  # Graphviz-5.0.0_skip-install-data-hook.patch
+]
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('binutils', '2.38'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+    ('SWIG', '4.0.2'),
+    ('pkgconf', '1.8.0'),
+]
+
+dependencies = [
+    ('Java', '17', '', True),
```

```suggestion
    ('Java', '11', '', True),
```
Does `Graphviz` also work with `Java/11`? The CI has this error:
```
found 2 variants of 'Java' dependency in easyconfigs using 'GCCcore-11.3' toolchain generation
* version: 11; versionsuffix:  as dep for set(['ANTLR-2.7.7-GCCcore-11.3.0-Java-11.eb', 'Bazel-5.1.1-GCCcore-11.3.0.eb'])
* version: 17; versionsuffix:  as dep for set(['Graphviz-5.0.0-GCCcore-11.3.0.eb', 'FORD-6.1.15-GCCcore-11.3.0.eb'])
```

If `Java/11` works, I would suggest to use this. Otherwise we would have to add an exception for the dependency.